### PR TITLE
[TIMOB-18822] Added check so that os name and powershell info is only de...

### PR DESCRIPTION
**Pull request by cb1kenobi**
...tected on Windows 8 or newer.

### CODE REVIEW

File: lib/env.js [hunk 1]
```diff
@@ -1,118 +1,120 @@
...
+				type: 'warning',
+				message: __('Windows Store apps are not supported on this version of Windows.') + '\n' +
+					__('You must use Windows 8 or newer to create Windows Store apps.')
+			});
+			emitter.emit('detected', results);
+			return callback(null, results);
+		}
+
+		async.series([
+			function osInfo(next) {
```
Comment (sgtcoolguy) [inline lib/env.js]: We may want to add a comment regarding why we're still doing this os info check now that we check the min version above using os.release().

It looks like this gives more accurate results for version and gives us the full name.

os.release() shows 6.2.9200 for me, while the info below gets 6.3.9600

**Comment by sgtcoolguy** [comment]
FR is good for Windows 8.1 Enterprise. Someone with Win 7 will need to do that FR.

File: lib/env.js (hunk 1 above)
Comment (cb1kenobi) [inline lib/env.js]: Wow, not sure why `os.release()` would return a different version. That's very strange.

Correct on the name... `wmic` will print the full operating system name so we can determine if it's "pro", "home premium", "enterprise", etc.

**Comment by eharrisappcel** [comment]
FR on Windows 7 looks good!

**Comment by sgtcoolguy** [comment]
@cb1kenobi Do we now need a PR for titanium_mobile to add new windowslib? I thinkw e also need this version bumped up and changelog updated

**Comment by cb1kenobi** [comment]
@sgtcoolguy Yes, we need a PR for timob. Did you publish windowslib?

**Comment by sgtcoolguy** [comment]
@cb1kenobi Pushed as 0.1.11

**Pull request merged**